docs/advanced-topics/checker-workflows.rst

```diff
@@ -105,12 +104,15 @@ Notice that the checker parameter file has the same content as the
 original parameter file, in addition to having a checker specific
 parameter.
 
-One point of confusion is that a checker workflow contains a validation
-tool/workflow. The validation tool/workflow is what does the bulk of the
-validation. It is responsible for ensuring that the results of the
-original entry match expected results. The checker workflow refers to
-the workflow that connects the original entry with the validation
-tool/workflow, so that it can be run as one workflow.
+Recall that the checker workflow refers to the workflow that connects
+the original entry with the validation tool/workflow, so that it can be
+run as one workflow. The validation tool/workflow is what does the bulk
+of the validation. It is responsible for ensuring that the results of the
+original entry match expected results. 
+
+To restate: First, the original workflow is run on a known input. Then, a
+validation workflow runs on the outputs of the original workflow, and makes
+sure that they are valid. Typically this means comparing a known md5sum.
```

Comment:
> Typically this means comparing a known md5sum.

Is this true? I don't know. It certainly is for your example, but can you generalize this for most checker workflows? Again, maybe the answer is yes, just checking.

Reply:
Maybe "often"?
Probably depends on how many algorithms are using random seeds and/or display output that changes over time, timestamps, dates, etc.

Reply:
I'm not aware of any checker workflows for algorithms involving random seeds. I reckon such a checker workflow would either just be running the actual workflow with known data and making sure it doesn't return !0, or would need to be custom-built around the output (such as running an R script to check for concordance within some tolerance threshold; I have done this before with a non-random alg that rounds differently on different backends) therefore outside the scope of an introduction like this.